scripts/touched.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import os
+import json
+
+query = """
+{
+  repository(name: "website", owner: "kubernetes") {
+    pullRequests(labels: "language/fr", last: 100) {
+      edges {
+        node {
+          title
+          state
+          url
+          files (last: 100) {
+            edges {
+              node {
+                path
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+"""
+
+r = requests.post("https://api.github.com/graphql",
+        json={"query": query},
+        headers={
+          "Authorization": "token %s" % os.environ["GITHUB_TOKEN"],
+          "Accept": "application/vnd.github.ocelot-preview+json",
+          "Accept-Encoding": "gzip"
+        })
+
+reply = r.json()
+prs = reply['data']['repository']['pullRequests']['edges']
+
+for pr in prs:
+  print(pr["node"]["title"])
+  print(pr["node"]["url"])
+
+  files = pr["node"]["files"]["edges"]
+  for f in files:
+    #import pdb
+    #pdb.set_trace()
```

Comment:
Possibly make `last` configurable?
Is the `state` field used?
nit: remove debug statements

Reply:
done